icedb/icedb.py

```diff
@@ -404,7 +416,7 @@ def tombstone_cleanup(self, min_age_ms: int) -> tuple[list[str], list[str], list
             1,
             schema,
             list(data_files_to_keep.values()),
-            None,
+            list(log_files_to_keep.values()),
```

Comment:
```suggestion
            list(log_files_to_keep.values()),  # kept because we must preserve tombstones for `min_age`
```
a comment on why would be good here